apps/token-e2e/project.json

```diff
@@ -21,6 +21,13 @@
       "options": {
         "lintFilePatterns": ["apps/token-e2e/**/*.{js,ts}"]
       }
+    },
+    "build": {
+      "executor": "@nrwl/workspace:run-commands",
+      "outputs": [],
+      "options": {
+        "command": "yarn tsc --project ./apps/trading-e2e/"
```

Comment:
Should all these be `./apps/trading-e2e/`?

Reply:
of course not, fix is on the way